src/steps/active-directory/__tests__/client.test.ts

```diff
@@ -0,0 +1,154 @@
+import { integrationConfig } from '../../../../test/config';
+import { Recording, setupProjectRecording } from '../../../../test/recording';
+import { Machine, UserLogon } from '../../../types';
+import { DefenderClient } from '../../ms-defender/clients/defenderClient';
+
+const logger: any = 'https://api.securitycenter.microsoft.com/api';
```

Comment:
It isn't clear what this is meant to be, since it is named `logger` but is a String. 🤔